src/components/CommentList.tsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import { Comment } from '../types/Comment';
+
+type Props = {
+  comments: Comment[];
+  onDelete: (id: number) => void;
+};
+
+export const CommentList: React.FC<Props> = ({ comments, onDelete }) => {
+  return (
+    <>
+      <p className="title is-4">Comments:</p>
```

Comment:
Consider using a more descriptive heading level for accessibility. Ensure that the heading level fits the overall document structure.